plugins/outputs/stomp/README.md

```diff
@@ -0,0 +1,31 @@
+# STOMP Producer Output Plugin
+
+This plugin writes to a [Active MQ Broker](http://activemq.apache.org/) Applying STOMP Protocol (https://stomp.github.io).
+
+It also support Amazon MQ  (https://aws.amazon.com/amazon-mq/)
+
+
+## CONFIGURATON
+
+```toml
+  ## Host of Active Mq broker
+    host = "localhost:61613"
+
+  ## Queue name for producer messages
+    queueName = "telegraf"
+
+  ## Optional username and password if Required to connect Active MQ server.
+    username = ""
+    password = ""
+
+  ## Optional TLS Config
+    # tls_ca = "/etc/telegraf/ca.pem"
+    # tls_cert = "/etc/telegraf/cert.pem"
+    # tls_key = "/etc/telegraf/key.pem"
+    ## Use TLS but skip chain & host verification
+    # insecure_skip_verify = false
+
+  ## Data format to output.
+    # data_format = "json"
+```
```

Comment:
Please fix the formatting here and comment sensible defaults as documented [here](https://github.com/influxdata/telegraf/blob/master/docs/developers/SAMPLE_CONFIG.md#style). Please also adapt the `sampleConfig` accordingly.
```suggestion
```toml
  ## Host of Active MQ broker
  host = "localhost:61613"

  ## Queue name for producer messages
  # queueName = "telegraf"

  ## Username and password if required by the Active MQ server.
  # username = ""
  # password = ""

  ## Optional TLS Config
  # tls_ca = "/etc/telegraf/ca.pem"
  # tls_cert = "/etc/telegraf/cert.pem"
  # tls_key = "/etc/telegraf/key.pem"
  ## Use TLS but skip chain & host verification
  # insecure_skip_verify = false

  ## Data format to output.
  # data_format = "json"
```
```